ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-chain.yaml

```diff
@@ -2,5 +2,6 @@ chain:
   as: cucushift-installer-check-cluster-health
   steps:
   - ref: cucushift-installer-check-cluster-health
+  - ref: cucushift-installer-check-control-plane-machine-name
```

Comment:
Will this check works on all profiles, e.g, sno, hypershift, rosa, aro, ...

Reply:
In the `cucushift-installer-check-control-plane-machine-name` step, I saw the following pre-check code:
```
oc get machine -n openshift-machine-api --selector machine.openshift.io/cluster-api-machine-type=master --no-headers -owide 1>${machine_output} 2>${err_output}

if grep -ir 'No resources found in openshift-machine-api namespace.' ${err_output}; then
    echo "WARN: No machines in openshift-machine-api namespace, skip checking"
    exit 0
fi
```
So I guess it should work in most of situations.

Reply:
Ack